internal/builders/maven/plugins/verification-plugin/src/main/java/io/github/adamkorcz/SlsaVerificationMojo.java

```diff
@@ -0,0 +1,189 @@
+            getLog().info("Skipping slsa verification: Fail to retrieve slsa-verifier.");
+            return;
+        }
+
+        // Verify the slsa of each dependency
+        Set<Artifact> dependencyArtifacts = project.getDependencyArtifacts();
+        for (Artifact artifact : dependencyArtifacts ) {
+            // Retrieve the dependency jar and its slsa file
+            String artifactStr = artifact.getGroupId() + ":" + artifact.getArtifactId() + ":" + artifact.getVersion();
+            try {
+                // Retrieve the slsa file of the artifact
+                executeMojo(
+                    plugin(
+                        groupId("com.googlecode.maven-download-plugin"),
+                        artifactId("download-maven-plugin"),
+                        version("1.7.0")
+                    ),
+                    goal("artifact"),
+                    configuration(
+                        element(name("outputDirectory"), "${project.build.directory}/slsa"),
+                        element(name("groupId"), artifact.getGroupId()),
+                        element(name("artifactId"), artifact.getArtifactId()),
+                        element(name("version"), artifact.getVersion()),
+                        element(name("type"), "intoto.build.slsa"),
+                        element(name("classifier"), "jar")
+                    ),
+                    executionEnvironment(
+                        project,
+                        mavenSession,
+                        pluginManager
+                    )
+                );
+
+                // Retrieve the dependency jar if slsa file does exists for this artifact
+                executeMojo(
+                    plugin(
+                        groupId("org.apache.maven.plugins"),
+                        artifactId("maven-dependency-plugin"),
+                        version("3.6.0")
+                    ),
+                    goal("copy"),
+                    configuration(
+                        element(name("outputDirectory"), "${project.build.directory}/slsa"),
+                        element(name("artifact"), artifactStr)
+                    ),
+                    executionEnvironment(
+                        project,
+                        mavenSession,
+                        pluginManager
+                    )
+                );
+            } catch(MojoExecutionException e) {
+                getLog().info("Skipping slsa verification for " + artifactStr + ": No slsa file found.");
+                continue;
+            }
+
+            // Verify slsa file
+            try {
+                // Run slsa verification on the artifact and print the result
+                // It will never fail the build process
+                String arguments = "verify-artifact --provenance-path ";
+                arguments += "${project.build.directory}/slsa/" + artifact.getArtifactId() + "-" + artifact.getVersion() + "-jar.intoto.build.slsa ";
```

Comment:
nit: remote intoto from the name. The builder at v1.70 does have this extension but it's been removed. v1.80 won't have it.